extensions/amp-date-countdown/validator-amp-date-countdown.protoascii

```diff
@@ -64,6 +64,9 @@ tags: {  # <amp-date-countdown>
     name: "offset-seconds"
     value_regex: "-?\\d+"
   }
+  # TODO(gregable): Implement validation that requires the template attr value
+  # to reference the id of an existing template element.
+  attrs: { name: "template" }
```

Comment:
@honeybadgerdontcare Validator change FYI.

Reply:
@choumx I'm not following the todo. Are you allowing an attribute that needs additional validation in the validator engine for it to work properly? Usually we wait to add that engine change before allowing the attribute. Or do we not care if that is enforced in validation and if it does get enforced later it is okay to invalidate pages that do it wrong?

Reply:
I copied this TODO from another extension that has the same feature: https://github.com/ampproject/amphtml/blob/617b7e4e25aa578756c5af941c2748c50abd8004/extensions/amp-list/validator-amp-list.protoascii#L74-L76

Reply:
This feature doesn't require additional validator changes to work properly. 

The comment describes a nice-to-have additional validation step. I imagine that we'd either make it a validator warning or error, depending on the # of pages affected.

Reply:
Thank you for the explanation. Filed b/116870484 for tracking.